Cleaner report on completed stages (#68)
onnx/turnkeyml: docs/release_notes.md

If you are creating the release notes for a new version, please see the [template](#template-version-majorminorpatch). Release notes should capture all of the significant changes since the last numbered package release.

# Version 1.1.0

This version focuses on improving the clarity of the telemetry reported.

## Users

### User Improvements

- Report splits `stages_completed` into stage status and duration.

## User Breaking Changes

None.

## Developers

### Developer Improvements

None

### Developer Breaking Changes

None

> 

Author: Do we really want to update this file every time the minor version changes? Looks like this could grow very quickly.

If you are creating the release notes for a new version, please see the [template](#template-version-majorminorpatch). Release notes should capture all of the significant changes since the last numbered package release.

# Version 1.1.0

This version focuses on improving the clarity of the telemetry reported.

## Users

### User Improvements

> - ONNX files exported from PyTorch models now have a `torch_export_verified` key in their stats/report files that indicates whether the `torch.onnx.verification.find_mismatch()` API could find any issue with the exported ONNX file.

Reviewer: The key is called `torch_export_validity`.
Other wise looks good.

- Stats and report CSV files split `stages_completed` into stage status and duration.
- Build, benchmark, and stage status values in the stat and report files now use the same terminology values:

```
class FunctionStatus(enum.Enum):

>     # INCOMPLETE indicates stage/build/benchmark is either running or was killed;

Author: Recommend moving this down close to ERROR and KILLED.
Please also highlight that all INCOMPLETE stages have been either KILLED or ERRORed out.

    # if you know the process ended then it was killed;

>     # if the process is still running, stage/build/benchmark is still running.

Author: This is quire confusing, especially line 27. Please wordsmith.

> class FunctionStatus(enum.Enum):
>     # INCOMPLETE indicates stage/build/benchmark is either running or was killed;
>     # if you know the process ended then it was killed;
>     # if the process is still running, stage/build/benchmark is still running.

Author: ```suggestion
class FunctionStatus(enum.Enum):
    # INCOMPLETE indicates that a stage/build/benchmark started and did not complete;
    # All Stages/builds/benchmarks are marked as INCOMPLETE upon start.
    # Stages/builds/benchmarks that started and were temporarily marked as INCOMPLETE
    # will become either SUCCESSFUL, ERROR, or KILLED.
```

Reviewer 2: Incorporated into some more wordsmithing

    INCOMPLETE = "incomplete"

>     # NOT_STARTED applies to stages that didnt start because
>     # the build errored out or was killed prior to stage starting.

Author: Previous sentence sounds like the condition here is `didn't start + reason for not starting`. What we actually want to say is `didn't start`, which may be caused by `reason`.
```suggestion
    # NOT_STARTED applies to stages that didn't. This happens when
    # the build errors out or when the build is killed prior to stage starting.
```

Reviewer 2: Incorporated into some more wordsmithing
